### PR TITLE
Fix/login bugs

### PR DESCRIPTION
**Pull request by marwanm7moud**

### CODE REVIEW

File: feature/authectication/presentation/ui/src/main/res/layout-land/fragment_login.xml [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+            app:layout_constraintTop_toBottomOf="@+id/textview_username">
+
+            <com.google.android.material.textfield.TextInputEditText
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:background="@drawable/edittext_background"
+                android:inputType="textPersonName"
+                android:maxLines="1"
+                android:maxLength="24"
+                android:padding="16dp"
```
Comment (mohammed-akkad) [inline feature/authectication/presentation/ui/src/main/res/layout-land/fragment_login.xml]: use dimen

File: feature/authectication/presentation/ui/src/main/res/xml/fragment_login_xml_landscape_constraintlayout_scene.xml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        <Constraint
+            motion:layout_constraintEnd_toStartOf="@+id/textview_signup"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            motion:layout_constraintHorizontal_chainStyle="packed"
+            motion:layout_constraintTop_toBottomOf="@+id/button_login"
+            android:layout_marginStart="@dimen/spacing_medium_16dp"
+            motion:layout_constraintHorizontal_bias="0.5"
+            motion:layout_constraintStart_toStartOf="parent"
+            android:layout_marginTop="16dp"
```
Comment (mohammed-akkad) [inline feature/authectication/presentation/ui/src/main/res/xml/fragment_login_xml_landscape_constraintlayout_scene.xml]: use dimen

**Review by mohammed-akkad — COMMENTED** [review]
great work

File: feature/authectication/presentation/ui/src/main/res/layout/fragment_login.xml [hunk 3]
```diff
@@ -109,6 +109,7 @@
                 android:background="@drawable/edittext_background"
                 android:inputType="textPersonName"
                 android:maxLines="1"
+                android:maxLength="24"
                 android:padding="16dp"
```
Comment (MohamedElgohary88) [inline feature/authectication/presentation/ui/src/main/res/layout/fragment_login.xml]: use dimens

File: feature/authectication/presentation/ui/src/main/res/xml/fragment_login_xml_landscape_constraintlayout_scene.xml [hunk 4]
```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="utf-8"?>
+<MotionScene xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:motion="http://schemas.android.com/apk/res-auto">
+
+    <Transition
+        android:id="@+id/show_keyboard"
+        motion:constraintSetEnd="@+id/end"
+        motion:constraintSetStart="@id/start"
+        motion:duration="1000">
```
Comment (mosayed01) [inline feature/authectication/presentation/ui/src/main/res/xml/fragment_login_xml_landscape_constraintlayout_scene.xml]: 300 is better

File: feature/authectication/presentation/ui/src/main/res/xml/fragment_login_xml_landscape_constraintlayout_scene.xml [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+        <OnClick motion:targetId="@id/textview_username"
+                motion:clickAction="transitionToEnd"
+
```
Comment (mosayed01) [inline feature/authectication/presentation/ui/src/main/res/xml/fragment_login_xml_landscape_constraintlayout_scene.xml]: white space

**Review by mosayed01 — CHANGES_REQUESTED** [review]

**Review by mosayed01 — APPROVED** [review]

**Pull request merged**